### PR TITLE
Add ability to use multiarch image for awscli

### PR DESCRIPTION
**Pull request by fbalak**
Signed-off-by: Filip Balak <fbalak@redhat.com>

### CODE REVIEW

File: ocs_ci/utility/utils.py [hunk 1]
```diff
@@ -2365,3 +2365,21 @@ def set_aws_region(region=None):
...
+def get_system_architecture():
+    """
+    Get output from ''uname -m' command run on first worker node.
+
+    Returns:
+        str: Architecture of system
+    """
+    from ocs_ci.ocs import ocp
+
+    node_obj = ocp.OCP(kind=constants.NODE)
```
Comment (AviadP) [inline ocs_ci/utility/utils.py:2379]: you can use ocs_ci/ocs/node.py::get_typed_nodes instead of this code block

File: ocs_ci/utility/utils.py [hunk 2]
```diff
@@ -2365,3 +2365,21 @@ def set_aws_region(region=None):
     log.debug("Exporting environment variable AWS_REGION")
     region = region or config.ENV_DATA['region']
     os.environ['AWS_REGION'] = region
+
+
+def get_system_architecture():
+    """
+    Get output from ''uname -m' command run on first worker node.
```
Comment (vavuthu) [inline ocs_ci/utility/utils.py:2372]:
```suggestion
    Get output from 'uname -m' command run on first worker node.
```

File: ocs_ci/utility/utils.py [hunk 3]
```diff
@@ -2365,3 +2365,21 @@ def set_aws_region(region=None):
...
+
+    Returns:
+        str: Architecture of system
```
Comment (vavuthu) [inline ocs_ci/utility/utils.py:2375]: leave empty line after this

**Comment by fbalak** [comment]
Verification failed for `tests.manage.rgw` and `test_add_node` tests. The rest of 58 tests from tier1 passed.

**Comment by fbalak** [comment]
Fixes https://github.com/red-hat-storage/ocs-ci/issues/2435

File: ocs_ci/utility/utils.py (hunk 3 above)
Comment (fbalak) [reply ocs_ci/utility/utils.py:2457]: Done

File: ocs_ci/utility/utils.py (hunk 2 above)
Comment (fbalak) [reply ocs_ci/utility/utils.py]: Done

File: ocs_ci/utility/utils.py (hunk 1 above)
Comment (fbalak) [reply ocs_ci/utility/utils.py]: Done

**Review by AviadP — APPROVED** [review]

**Review by RazTamir — APPROVED** [review]

**Pull request merged**